packages/styles/scss/ibm-dotcom-styles.scss

```diff
@@ -35,30 +35,30 @@
 @import 'components/video-player/video-player';
 
 // Patterns - Sections
```

Comment:
We can get rid of the patterns - sections and patterns - blocks categorization in this file